gammapy/modeling/models/tests/test_cube.py

```diff
@@ -284,10 +295,16 @@ def test_evaluate_array(sky_model):
         lon = 3 * u.deg * np.ones(shape=(3, 4))
         lat = 4 * u.deg * np.ones(shape=(3, 4))
         energy = [1, 1, 1, 1, 1] * u.TeV
+        times = Time([10, 20], format="mjd")
 
-        q = sky_model.evaluate(lon, lat, energy[:, np.newaxis, np.newaxis])
+        q = sky_model.evaluate(
+            lon,
+            lat,
+            energy[:, np.newaxis, np.newaxis],
+            times[:, np.newaxis, np.newaxis, np.newaxis],
+        )
 
-        assert q.shape == (5, 3, 4)
+        assert q.shape == (2, 5, 3, 4)
```

Comment:
As discussed in the call today, the evaluation of the temporal model does not change the shape of the flux array, but just computes the norm in the given time interval. Maybe you can adjust the test here accordingly, to avoid confusion.